vrt.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.license       = 'MIT'
   spec.files         = Dir['lib/**/*.{rb,json}']
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '>= 2.5'
+  spec.required_ruby_version = '~> 2.5'
```

Comment:
haha we've talked about this so many times so please don't kill me, but this'll make it `2.5.x` basically right?